src/GraphQL/FieldHelper/AbstractFieldHelper.php

```diff
@@ -115,15 +111,16 @@ public function getArguments(FieldNode $ast)
      */
     public function extractData(&$data = [], $container, $args, $context, ResolveInfo $resolveInfo = null)
     {
-        if ($container instanceof ElementInterface) {
+        if (method_exists($container, 'getId')) {
```

Comment:
why ?

Reply:
That might have been an artifact of a problem I had when `extractData()` got a `Hotspotimage` object which was an `ElementInterface`, I solved it by sending `$relation->getImage()` rather than `$relation` in `Helper\ImageGallery`. I should restore it I think.